docs/02-guides/extend-vite-configuration/mdx.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: MDX
+description:
+  This guide explains how to setup MDX in your Front-Commerce application thanks
+  to Vite.
+---
+
+<p>{frontMatter.description}</p>
+
+:::tip
+
+You can check the
+[`example-extension/mdx-blog`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/tree/main/skeleton/example-extensions/mdx-blog)
+example to see how MDX can be implemented.
+
+:::
+
+## Setup
+
+### Install MDX bundler for vite
+
+```sh
+pnpm add @mdx-js/rollup @mdx-js/react
+```
+
+### Update `vite.config.ts`
+
+**Please note that `mdx` plugin must be declared before `frontCommerce`**
```

Comment:
**nitpick:** without explanations, it sounds a bit :shrug: 

**suggestion:** explain the reason for this constraint.

Example (**note:** I'm not sure about the real reason… you know it better than I do):
```suggestion
**Please note that `mdx` plugin must be declared before the `frontCommerce` plugin** to ensure that MDX files have already been transformed to standard React components before going through the Front-Commerce–Remix–React toolchain.
```

Reply:
That's it